kafka_helper/__init__.py

```diff
@@ -121,7 +121,7 @@ def get_kafka_producer(
 
 def get_kafka_consumer(
         topic: T.Optional[str] = None,
-        value_deserializer: T.Callable[[bytes], _T] = (
+        value_deserializer: T.Optional[T.Callable[[bytes], _T]] = (
             lambda v: json.loads(v.decode('utf-8'))
             ),
```

Comment:
Actually, just make this `value_deserializer: T.Optional[T.Callable[[bytes], bytes]] = None`. I was gonna try to make `ConsumerRecord` generic on its `value` (which would match the return type of `value_deserializer`), but unfortunately `ConsumerRecord` is a `NamedTuple`, and Mypy doesn’t support generic `NamedTuple`s. Also change the signature in `marketview_etl/stubs/kafka/consumer/__init__.pyi` to match. This effectively hard-codes the generic to the default (`bytes`), and forces it to be so.